src/Sulu/Bundle/EventLogBundle/Application/Collector/DomainEventCollectorInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\EventLogBundle\Application\Collector;
+
+use Sulu\Bundle\EventLogBundle\Domain\Event\DomainEvent;
+
+interface DomainEventCollectorInterface
+{
+    public function collect(DomainEvent $domainEvent): void;
+
+    public function clear(): void;
+
+    public function dispatch(): void;
```

Comment:
should we be here more specific to avoid confusen e.g.: `dispatchCollectedEvents` what do you think? @nnatter

Reply:
would not use a `CollectedEvents` suffix, we would need to do that for the `clear` method too and it feels cumbersome to me. dont think `dispatch` is so bad because the method does not have any arguments in contrast du the `EventDispatcher::dispatch`. but `flush` would be okay for me too, do you like that better?

Reply:
`release` ??

Reply:
In the repository we did use `commit` as I remember should we go with that here also? But if we can also stay with `dispatch` as you are right it doesn't take any arguments so it should not be too much a problem.

Reply:
Isn't about "collecting" events and "releasing" or "dispatching" them in future?

Reply:
`commit` feels like i am persisting/commiting some changes.`release` feels like i am setting something free. i think i would expect that a `release` method returns the events and i can handle them by myself, not that it dispatches the events. but dont have a too strong opinion, all of `dispatch`/`release`/`flush` are okay for me 🙂

Reply:
Then I would stay with `dispatch`. As mention once I personally not a fan of that doctrine uses the word `flush` for what is happening there and `release` also sounds similar to this for me.

Reply:
`dispatch` +1